docs/system/container-metrics.md

```diff
@@ -0,0 +1,108 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.time(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.time` | Counter | `ns` | Total CPU time consumed [1] |
+
+**[1]:** Total CPU time consumed by the specific container.
```

Comment:
I think we should also mention that it's not constrained to one core.

Reply:
At https://github.com/open-telemetry/semantic-conventions/pull/282#issuecomment-1909845087 I mentioned `I have left out the container.cpu.logical_number for now so as to start small with this PR`. Does it make sense to make this statement without this attribute at this point 🤔 ?

Reply:
updated: [92ddfe7](https://github.com/open-telemetry/semantic-conventions/pull/282/commits/92ddfe76e633a5c9151f1dc8483e5fb231725518)